src/locale/mr.js

```diff
@@ -52,19 +89,19 @@ export default moment.defineLocale('mr', {
         sameElse : 'L'
     },
     relativeTime : {
-        future : '%s नंतर',
-        past : '%s पूर्वी',
-        s : 'सेकंद',
-        m: 'एक मिनिट',
-        mm: '%d मिनिटे',
-        h : 'एक तास',
-        hh : '%d तास',
-        d : 'एक दिवस',
-        dd : '%d दिवस',
-        M : 'एक महिना',
-        MM : '%d महिने',
-        y : 'एक वर्ष',
-        yy : '%d वर्षे'
+        future: '%sमध्ये',
```

Comment:
I agree that Marathi is super dynamic :).
Even in the case of '%sत', it is a short form of '%sच्या आत'. As in:
```
काही सेकंदांच्या आत
%d मिनिटांच्या आत
%d तासांच्या आत
%d दिवसांच्या आत
%d महिन्यांच्या आत
%d वर्षांच्या आत
```
And this is with अनुस्वार, even though the short form is without अनुस्वार (afaik).
But no one uses that full form these days. :)